disnake/ext/commands/interaction_bot_base.py

```diff
@@ -139,6 +146,21 @@ def _format_diff(diff: _Diff) -> str:
     return "\n".join(f"| {line}" for line in lines)
 
 
+def _match_subcommand_chain(
+    command: InvokableSlashCommand, chain: List[str]
+) -> Union[InvokableSlashCommand, SubCommand, SubCommandGroup, None]:
+    if command.name != chain[0]:
+        return None
+    if len(chain) == 1:
+        return command
+    if len(chain) == 2:
+        return command.children.get(chain[1])
+    if len(chain) == 3:
+        group = command.children.get(chain[1])
+        if isinstance(group, SubCommandGroup):
+            return group.children.get(chain[2])
```

Comment:
It'd be nice if there was some form of documentation here to describe what this function achieves without the need for external context

Reply:
https://github.com/DisnakeDev/disnake/pull/1107/commits/e86e5cb646a2c3ba99c2bb0318acd3f2fec51572